<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="WINCTRL-S7E21V6" name="Regression suite 1" tests="14" failures="2" timestamp="15 Jun 2017 02:18:09 GMT" time="0.09" errors="0">
  <testcase name="login" time="0.013" classname="TestNGDemos.TestNGDemo2">
    <failure type="java.lang.ArithmeticException" message="/ by zero">
      <![CDATA[java.lang.ArithmeticException: / by zero
	at TestNGDemos.TestNGDemo2.login(TestNGDemo2.java:18)
... Removed 24 stack frames]]>
    </failure>
  </testcase> <!-- login -->
  <testcase name="customerDetails" time="0.0" classname="TestNGDemos.TestNGDemo2">
    <skipped/>
  </testcase> <!-- customerDetails -->
  <testcase name="funA" time="0.003" classname="TestNGDemos.TestNGDemo3"/>
  <testcase name="funC" time="0.003" classname="TestNGDemos.TestNGDemo3"/>
  <testcase name="funD" time="0.002" classname="TestNGDemos.TestNGDemo3"/>
  <testcase name="funB" time="0.001" classname="TestNGDemos.TestNGDemo3"/>
  <testcase name="funA" time="0.002" classname="TestNGDemos.TestNGGroupsDemo"/>
  <testcase name="funB" time="0.002" classname="TestNGDemos.TestNGGroupsDemo"/>
  <testcase name="funC" time="0.002" classname="TestNGDemos.TestNGGroupsDemo"/>
  <testcase name="funD" time="0.002" classname="TestNGDemos.TestNGGroupsDemo"/>
  <testcase name="funB" time="0.001" classname="TestNGDemos.TestNGSkipDemo">
    <failure type="java.lang.ArithmeticException" message="/ by zero">
      <![CDATA[java.lang.ArithmeticException: / by zero
	at TestNGDemos.TestNGSkipDemo.funB(TestNGSkipDemo.java:26)
... Removed 24 stack frames]]>
    </failure>
  </testcase> <!-- funB -->
  <testcase name="funD" time="0.001" classname="TestNGDemos.TestNGSkipDemo">
    <skipped/>
  </testcase> <!-- funD -->
  <testcase name="funA" time="0.001" classname="TestNGDemos.TestNGSkipDemo">
    <skipped/>
  </testcase> <!-- funA -->
  <testcase name="f" time="0.002" classname="TestNGDemos.TestNGXML2"/>
</testsuite> <!-- Regression suite 1 -->
